language-plutus-core/language-plutus-core.cabal

```diff
@@ -78,7 +85,7 @@ library
         Data.Functor.Foldable.Monadic
         Data.Text.Prettyprint.Doc.Custom
     default-language: Haskell2010
-    default-extensions: ExplicitForAll ScopedTypeVariables
+    default-extensions: ExplicitForAll ScopedTypeVariables FlexibleContexts
```

Comment:
I needed to add `FlexibleContexts` to `default-extensions` in order to replace `PrettyCfg` by `PrettyPlc` everywhere without also adding that extension to the top of each file where `PrettyCfg` was used. If that's not desired, I can revert the change, but I think it's a benign extension.

Reply:
I considered this and `FlexibleInstances` when I added the `default-extensions`, but I recall that there were objections.

Reply:
It's mostly harmless, but not entirely, and I'd like to make it explicit in any case.

Reply:
> It's mostly harmless, but not entirely

What is not harmless about `FlexibleContexts`?

Reply:
Oh nevermind, it seems to be fine, but it should still be explicit as it is a type system extension.